### ISSUE TITLE
Clicking a link in the pdf performs a page change but does not emit a (pageChange) event

### ISSUE THREAD

**Issue by NeuralNett** [VadimDez/ng2-pdf-viewer#474]
##### Bug Report or Feature Request (mark with an `x`)
```
- [ ] Regression (a behavior that used to work and stopped working in a new release)
- [X] Bug report -> please search issues before submitting
- [ ] Feature request
- [ ] Documentation issue or request
```
Hi there, i'm trying to keep track of the new page number after using in-document links with the following configuration:

`[src]="pdfSource"
[(page)]="currentPage"
(after-load-complete)="pdfLoaded($event)"
(pageChange)="pdfPageChange($event)"
[show-all]="false"
[render-text]="true"
[original-size]="false"
[fit-to-page]="true"`

Clicking a link in the pdf document jumps to the correct page, but does not emit a (pageChange) event and doesn't seem to update the [(page)] either.
(page-rendered) emits events only on initial page render, but not if a page is revisited.

Would you be able to extend the (pageChange) event emitter to capture link clicks?

**Comment by marabu-edv** [comment]
Hello, do you intend to implement this enhancement in the near future?

**Comment by VadimDez** [comment]
Released in `5.3.3`
